pages/index.js

```diff
@@ -221,8 +220,8 @@ const home = ({ session }) => {
                   </div>
                 </div>
                 <div className="mt-6">
-                  <nav className="grid gap-y-8">
-                    {mobileMenu.map((item) => (
+                  <nav className="grid gap-y-6">
+                    {mobileMenu.map((item)=>(
```

Comment:
I have kept this mobileMenu array(with single item) it would be helpful if more elements needs to be added in future.
Let me know if this needs to be removed

Reply:
It's ok to keep the mobile menu. But we should clone the same as the normalMenu

Reply:
Where would we use normalMenu? Currently there is no need for that.